src/components/AddMovies2.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+class AddMovies2 extends React.Component {
+  render() {
+    const { rating, genre, handleClick, handleChange, storyline } = this.props;
+    return (
+      <div>
+        <label data-testid="storyline-input-label" htmlFor="storyline">
+          Sinopse
+          <textarea
+            data-testid="storyline-input"
+            type="text"
+            id="storyline"
+            value={ storyline }
+            onChange={ this.handleChange }
```

Comment:
aqui você não deve utilizar o this
```suggestion
            onChange={ handleChange }
```